django/core/checks/messages.py

```diff
@@ -9,7 +9,8 @@
 class CheckMessage:
 
     def __init__(self, level, msg, hint=None, obj=None, id=None):
-        assert isinstance(level, int), "The first argument should be level."
+        if not isinstance(level, int):
+            raise TypeError("The first argument should be level.")
```

Comment:
We probably should update the error message too.
Something along the lines of level must be of type int?

Reply:
The current message sounds good to me.